lidar_navigation_cdt/include/grid_map_cdt/Challenge.hpp

```diff
@@ -14,7 +14,7 @@
 
 #include <grid_map_ros/grid_map_ros.hpp>
 
-#include <filters/filter_chain.h>
+#include <filters/filter_chain.hpp>
```

Comment:
It doesn't build due to this - we need to change back to `.h` for it to compile on both Melodic and Noetic

Reply:
there is the same issue in navigation_drs

Reply:
I will fix it

Reply:
Would you like me to do the changes in both or are you handling it?